configs/suse.tests/sles15/products.cfg

```
@@ -0,0 +1,4 @@
+[tool:pytest]
+addopts = --tb=short
+IMAGE = registry.mgr.suse.de/toaster-sles15-products
+TAGS = sles sles15 products salt-2019.2.2 python3
```

Since we're already using Salt 3000 in `products`, let's use the right tag.

```suggestion
TAGS = sles sles15 products salt-3000 python3
```